src/instructlab/cli/data/generate.py

```diff
@@ -279,6 +279,7 @@ def generate(
             system_prompt,
             legacy_pretraining_format,
         )
+        click.echo("✅ Data generate completed successfully!")
```

Comment:
Any concerns about character compatibility with the ✅ across terminals or other supported environments?

Reply:
re-considering whether can add a simple check first.